src/tools/papi-connector/README.md

```diff
@@ -1,10 +1,40 @@
-### Build papi connector for kokkos
+## Using the PAPI Connector
 
+To use the PAPI connector you have to compile it against the latest PAPI version that includes the new high-level API.
+
+#### Install PAPI (from version 6.0) or latest git version
```

Comment:
When do you anticipate version 6.0 will be tagged?
Would you please add a reference telling where one could download a stable version of Papi?